sql-schema/127.sql

```diff
@@ -0,0 +1,11 @@
+DROP TABLE IF EXISTS `ipmi_sensors`;
```

Comment:
Hence why I said components or static array, components is a central table system you can add data to rather than creating separate tables for everything.